action.yml

```diff
@@ -33,8 +33,21 @@ inputs:
   config_variables:
     description: 'Configuration variables to set on deploy'
     required: False
+  docker_repository_url:
+    description: 'URL for docker image repository'
+    required: False
+  auth_root_url:
+    description: 'Aptible Auth API endpoint'
+    required: False
+    default: https://auth.aptible.com
+  api_root_url:
```

Comment:
we should be consistent with setting defaults on these even if we're only using the auth one directly just to ease confusion